Lib/test/test_cprofile.py

```diff
@@ -139,6 +139,16 @@ def test_throw(self):
                 self.assertEqual(cc, 1)
                 self.assertEqual(nc, 1)
 
+    def test_bad_descriptor(self):
+        """
```

Comment:
I know that this is used above, but I am not a fan of docstrings in unittests :( Docstrings can be shown during test execution.

Let's maybe convert it to comments?

Reply:
Okay I converted this to a comment - I don't have a strong opinion on this (it's easier to do a docstring if the description is long so I normally just do a docstring). If you believe this actually matters in some cases, we should probably discuss it and put it to somewhere in devguide. It would be helpful for people like us to make a decision and get use to it :)